Traceback (most recent call last):
  File "d:\software\python3.7.2\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\software\python3.7.2\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\software\python3.7.2\Scripts\ampy.exe\__main__.py", line 9, in <module>
  File "C:\Users\Administrator.SC-201901111137\AppData\Roaming\Python\Python37\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\Administrator.SC-201901111137\AppData\Roaming\Python\Python37\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "C:\Users\Administrator.SC-201901111137\AppData\Roaming\Python\Python37\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\Administrator.SC-201901111137\AppData\Roaming\Python\Python37\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\Administrator.SC-201901111137\AppData\Roaming\Python\Python37\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "d:\software\python3.7.2\lib\site-packages\ampy\cli.py", line 337, in run
    output = board_files.run(local_file, not no_output)
  File "d:\software\python3.7.2\lib\site-packages\ampy\files.py", line 303, in run
    out = self._pyboard.execfile(filename)
  File "d:\software\python3.7.2\lib\site-packages\ampy\pyboard.py", line 273, in execfile
    return self.exec_(pyfile)
  File "d:\software\python3.7.2\lib\site-packages\ampy\pyboard.py", line 267, in exec_
    raise PyboardError('exception', ret, ret_err)
ampy.pyboard.PyboardError: ('exception', b"active: None
isconnected: True
conn:None
('192.168.3.4', '255.255.255.0', '192.168.3.1', '192.168.3.1')
('192.168.3.21', 1883)
", b'Traceback (most recent call last):
File "<stdin>", line 21, in <module>
File "umqtt/simple.py", line 59, in connect
OSError: [Errno 103] ECONNABORTED
')